flexget/plugins/input/smart_imdb.py

```diff
@@ -0,0 +1,478 @@
+from __future__ import unicode_literals, division, absolute_import
+
+import logging
+import re
+from jsonschema.compat import str_types
+from flexget import plugin
+from flexget.event import event
+from flexget.entry import Entry
+
+from imdb import IMDb
```

Comment:
This will throw an exception if imdbpy is not installed? Should be handled with a try-catch block imo.